# Professional Summary
Bioinformatician specializing in NGS-based miRNA and RNA-seq data analysis, with hands-on experience in reproducible workflow development using Snakemake, Python, R, and Bash. Skilled in automating end-to-end pipelines for preprocessing, differential expression, and functional enrichment analysis. Experienced in international collaborative research environments, contributing to cancer biomarker discovery and large-scale meta-analyses. Proficient in cloud-based computing (AWS) and statistical data interpretation, with a strong ability to extract meaningful biological insights from complex multi-omics datasets in genomics, transcriptomics, and microbiome profiling.

# Technical Skills & Tools
### Programming: Python, R, Bash, SQL, C++
### Workflow Managment: Snakemake
### Bioinformatics: NGS, RNA-seq, miRNA-seq, Whole Genome-Seq, Genomics, Transcriptomics, Comparative Genomics, phylogenetic analysis, Data Visualization
### Statistics: Statistical Analysis, Data Interpretation

# Selected Projects
### Hepatocellular carcinoma miRNA Meta-analysis
Analysis of miRNA NGS data from 7 studies to identify biomarkers in hepatocellular carcinoma. 
Performed quality control, trimming, alignment, quantification, differential expression, and functional enrichment analyses.

**Tools & Software:**  
- Quality control & trimming: FastQC, fastp  
- Alignment & quantification: miRDeep2  
- Differential expression: DESeq2 (R)  
- Functional enrichment: clusterProfilerÙˆ multiMiR  (R)  
- Workflow automation: Snakemake  
- Programming: Python, R, Bash

### Genes Responsible for Lactic Acid and Polysaccharide Production
A comparative genomics study of 26 *Lacticaseibacillus* genomes aimed at identifying which species possess the operons (LDH and EpsG) responsible for lactic acid and exopolysaccharide (EPS) production.  
Performed gene prediction, annotation, operon comparison, and phylogenetic analysis to determine lactic acid and EPS-producing strains.

**Tools & Software:**  
- Genome retrieval: NCBI GenBank  
- Gene prediction: Prodigal (v2.6.3)  
- Annotation: RAST (v2.0)  
- Comparative and operon analysis: BioCyc (v27.0)  
- Phylogenetic analysis: MEGA (v11.0)  
- Alignment & filtering: BioCyc multi-genome alignment  
- Languages/Environment: Python, R, Bash, Conda, Git, Linux

**Focus:**  
Functional genomics to identify *Lacticaseibacillus* species capable of producing lactic acid and polysaccharides, contributing to sustainable microbial biotechnology applications integrating both wet-lab and bioinformatics analyses.


